docs/pages/machine-id/guides/databases.mdx

```diff
@@ -0,0 +1,285 @@
+`tctl bots add ...`.
+
+Machine ID also allows you to use Linux ACLs to control access to certificates
+on disk. You will use this to ensure only your application has access to the
+short-lived certificates Machine ID uses.
+
+We'll work with the assumption you will be running Machine ID as the Linux user
+`teleport` and your application as the Linux user `app`. Create and initialize
+the destination directory by running this `tbot init` command either as `root`
+or as the `teleport` user:
+
+```code
+$ tbot init \
+    -c /etc/tbot.yaml \
+    --init-dir=/opt/machine-id \
+    --bot-user=teleport \
+    --owner=teleport:teleport \
+    --reader-user=app
+```
+
+<Admonition type="note" title="`tbot init` Note">
+Be sure to re-run `tbot init ...` as shown here if config templates are added
+or removed from `tbot.yaml`. You may run into permissions errors if `tbot init`
+is not run for new files.
+</Admonition>
+
+Next, you will use systemd to run Machine ID in the background on your
+application node. Create a systemd.unit file at
+`/etc/systemd/system/machine-id.service`:
+
+```systemd
+[Unit]
+Description=Teleport Machine ID Service
+After=network.target
+
+[Service]
+Type=simple
+User=teleport
+Group=teleport
+Restart=on-failure
+ExecStart=/usr/local/bin/tbot start -c /etc/tbot.yaml
+ExecReload=/bin/kill -HUP $MAINPID
+PIDFile=/run/machine-id.pid
+LimitNOFILE=8192
+
+[Install]
+WantedBy=multi-user.target
+```
+
+Additionally, we'll need to create a secondary service to manage the database
+proxy. Create another unit file at `/etc/systemd/system/machine-id-proxy.service`:
+
+```systemd
+[Unit]
+Description=Teleport Machine ID Proxy Service
+After=network.target
+Requires=machine-id.service
+
+[Service]
+Type=simple
+User=teleport
+Group=teleport
+Restart=on-failure
+ExecStart=/usr/local/bin/tbot -c /etc/tbot.yaml proxy --proxy=proxy.example.com:3080 --destination-dir=/opt/machine-id db --port=12345 example-server
+ExecReload=/bin/kill -HUP $MAINPID
+PIDFile=/run/machine-id-proxy.pid
+LimitNOFILE=8192
+
+[Install]
+WantedBy=multi-user.target
+```
+
+This will start a local proxy on port `12345` at which you may connect to the
+`example-server` database server. Be sure to customize the `tbot` parameters as
+necessary for your local setup.
+
+Finally, run the following commands to start Machine ID:
+
+```code
+$ sudo systemctl daemon-reload 
+$ sudo systemctl start machine-id machine-id-proxy
+$ sudo systemctl status machine-id machine-id-proxy
+```
+
+## Step 3/3. Update and run your application
+
+In the default proxy mode, database clients must also be configured to use
+`tbot`'s generated TLS certificates. This ensures no other users of the system
+can access the database via the local proxy, and ensures the connection between
+your database client and server is never unencrypted, even over localhost.
+
+The standard TLS credentials may be found in your configured destination
```

Comment:
Potentially worth mentioning the path to the key as well (even if it's literally just `/opt/machine-id/key`)

Reply:
good catch, will add!